scripts/ci/test-operator

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+# This script transforms an operator dir structured in the style of
+# community-operators into one expected by operator-registry, inserts the
```

Comment:
I believe `operator-courier` has implemented this functionality into it's `nest` command as a way to solve this exact problem. If possible, this should be ported to use the courier.